src/theory/strings/regexp_operation.cpp

```diff
@@ -117,161 +117,146 @@ RegExpConstType RegExpOpr::getRegExpConstType(Node r)
 
 // 0-unknown, 1-yes, 2-no
 int RegExpOpr::delta( Node r, Node &exp ) {
-  Trace("regexp-delta") << "RegExp-Delta starts with /" << mkString( r ) << "/" << std::endl;
+  std::map<Node, std::pair<int, Node> >::iterator itd = d_delta_cache.find(r);
```

Comment:
```suggestion
  auto itd = d_delta_cache.find(r);
```
And if you really don't want `auto`, I'd suggest to at least use a `const_iterator`.